resilience4j-circuitbreaker/src/main/java/io/github/resilience4j/circuitbreaker/CircuitBreakerConfig.java

```diff
@@ -169,6 +171,10 @@ public boolean isWritableStackTraceEnabled() {
         return writableStackTraceEnabled;
     }
 
+    public boolean isIgnoreUnknownExceptionsEnabled() {
+        return ignoreUnknownExceptions;
```

Comment:
Let's rename it ignoreClassBindingExceptions or ignoreClassConversationExceptions.
Otherwise people thing it means the property is used to configure that the CircuitBreaker would ignore any unknown exception which is not the case.